src/data/repositories/Dhis2ConfigRepository.ts

```diff
@@ -46,55 +48,61 @@ export class Dhis2ConfigRepository implements ConfigRepository {
         };
     }
 
-    public async getUiLocale(d2User: { settings: { keyUiLocale: string; keyDbLocale: string } }): Promise<string> {
```

Comment:
I see that you removed this function because was no longer being used in the project, right? in that case we can also remove `settings: { keyUiLocale: true }` from the getUser method